aspnetcore/signalr/groups.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Manage users and groups in SignalR
+author: 
+description: 
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: 
+ms.custom: mvc
+ms.date: 06/04/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/groups
+---
+
+# Manage users and groups in SignalR
+
+## Users?
+
+A user is one or more connections associated with an identifier. By default SignalR will use the `ClaimTypes.NameIdentifier` from the `ClaimsPrincipal` associated with the connection. There can be multiple connections for a single user. For example you can be connected on your desktop, your phone, and your laptop. If you send to the user all three connections will receive the message.
+
+You can send a message to a specific user by passing the user identifier to the `User(...)` function in your hub method as shown in the following example. One important aspect to note is that the user identifier is case-sensitive.
```

Comment:
Link to API docs! Check if there's a way to do that "semantically" (i.e. other than just grabbing a URL like: https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.signalr.hubcallerclients.user?view=aspnetcore-2.1)

Reply:
~~One important aspect to note is that~~ **Note:** The user identifier is case-sensitive

Also I think there's a way to have a note call-out in the special markdown syntax docs uses.

Reply:
It's:

> [!NOTE]
> note text note text note text